packages/common/src/sites/feeds/_internal/templates/dcatAP.ts

```diff
@@ -0,0 +1,195 @@
+        "{{ report.measResult.ConResult.conSpec.resTitle }}",
+      "foaf:page": {
+        "foaf:Document": "{{ metadata-url }}",
+        "dct:title": "Source metadata",
+        "@type": "foaf:Document",
+      },
+      "dcat:contactPoint": {
+        "vcard:org":
+          "{{ metadata.metadata.dataIdInfo.idPoC.rpOrgName || metadata.metadata.dataIdInfo.idCitation.citRespParty.rpOrgName || metadata.metadata.mdContact.rpOrgName || orgName || '' }}",
+        "vcard:hasEmail":
+          "{{  metadata.metadata.dataIdInfo.idPoC.rpCntInfo.cntAddress.eMailAdd || metadata.metadata.dataIdInfo.citRespParty.rpCntInfo.cntAddress.eMailAdd || metadata.metadata.mdContact.rpCntInfo.cntAddress.eMailAdd || orgContactEmail }}",
+        "@id": "{{ ownerUri }}",
+        "vcard:role":
+          "{{ metadata.metadata.dataIdInfo.idPoC.role.RoleCd.@_value || metadata.metadata.dataIdInfo.idCitation.citRespParty.role.RoleCd.@_value }}",
+        "vcard:URL":
+          "{{ metadata.metadata.dataIdInfo.idPoC.rpCntInfo.cntOnlineRes.linkage  || metadata.metadata.dataIdInfo.citRespParty.rpCntInfo.cntOnlineRes.linkages }}",
+        "@type": "vcard:Kind",
+      },
+      "dct:publisher": {
+        "@type": "foaf:Agent",
+        "foaf:org": "{{ data.attributes.orgName }}",
+      },
+      "dct:location": {
+        "@type": "dcat:bbox",
+        "@value":
+          "data.attributes.metadata.dataIdInfo.dataExt.geoEle.GeoBndBox || item.extent",
+      },
+      "dct:accessRights": {
+        "dct:rightsStatement":
+          "{{ metadata.dataIdInfo.resConst.SecConsts.class.ClasscationCd.@_value }}",
+      },
+      "dcat:Relationship": {
+        "dct:relation": {
+          "dct:isPartOf": "{{ base-item-url }}",
+        },
+      },
+      "dct:temporal": {
+        "@type": "dcterms:PeriodOfTime",
+        "dct:startDate": {
+          "@type":
+            "{{ data.attributes.metadata.metadata.dataIdInfo.dataExt.tempEle.TempExtent.exTemp.TM_Period.tmBegin }}",
+        },
+        "dct:endDate": {
+          "@value":
+            "{{ data.attributes.metadata.metadata.dataIdInfo.dataExt.tempEle.TempExtent.exTemp.TM_Period.tmEnd }}",
+          "@type": "xsd:date",
+        },
+      },
+      "dct:subject": "{{ data.attributes.categories }}",
+      "dct:provenance": {
+        "@type": "dct:ProvenanceStatement",
+        "@label": {
+          "@value": "{{ dataIdInfo.idCredit }}",
+        },
+      },
+      "dct:issued": {
+        "@value":
+          "{{ metadata.metadata.dataIdInfo.idCitation.date.pubDate || metadata.metadata.dataIdInfo.idCitation.date.createDate || created }}",
+        "@type": "xsd:dateTime",
+      },
+      "dct:modified": {
+        "@value": "{{ modified }}",
+        "@type": "xsd:dateTime",
+      },
+      "dct:Frequency": "{{ metadata.resMaint.maintFreq.@_value }}",
+      "dct:language": {
+        "@id": "{{ metadata.mdLang.countryCode.value “-” languageCode.value }}",
+      },
+      "dcat:distribution": [
```

Comment:
This is the one field in the dataset template that we _don't_ want to expose for configuration. Other feeds (dcat-us 1.1, dcat-ap 2.0.1, etc.) dynamically calculate the values of these distributions at the api level.

If a user manually goes in and adds `"dcat:distribution": [...]`, then we throw them a bone and append their custom distributions to our default ones. However, we don't advertise the functionality.